frontend/src/scenes/data-management/database/databaseTableListLogic.ts

```diff
@@ -67,5 +72,9 @@ export const databaseTableListLogic = kea<databaseTableListLogicType>([
                 })),
         ],
     }),
-    afterMount(({ actions }) => actions.loadDatabase()),
+    afterMount(({ actions, values }) => {
+        if (values.featureFlags[FEATURE_FLAGS.DATA_WAREHOUSE]) {
```

Comment:
Not super happy having to do this in two places... feels a bit whack-a-mole.

Didn't see another option because of the way the component mounts the logic (see https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow.tsx#L158)

Maybe @EDsCODE has more context on how that could be moved somewhere else we're sure has access to the datawarehouse. Or some more domain context to set dataware house up in test so that this endpoint doesn't return an error (which causes the toast)